gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcDirectServerStreamingCallable.java

```diff
@@ -70,4 +78,110 @@ public void serverStreamingCall(
     ClientCall<RequestT, ResponseT> call = GrpcClientCalls.newCall(descriptor, context);
     return ClientCalls.blockingServerStreamingCall(call, request);
   }
+
+  /**
+   * Wraps a GRPC ClientCall in a {@link StreamController}. It feeds events to a {@link
+   * ResponseObserver} and allows for back pressure.
+   */
+  static class GrpcStreamController<RequestT, ResponseT> extends StreamController {
+    private final ClientCall<RequestT, ResponseT> clientCall;
+    private final ResponseObserver<ResponseT> observer;
```

Comment:
I think naming the variable `responseObserver` would make the code clearer.

Reply:
done